Changes CHPL_MEM, and CHPL_TASK defaults to cstdlib and fifo, respectively
--- a/util/chplenv/chpl_mem.py
+++ b/util/chplenv/chpl_mem.py
@@ -27,7 +27,7 @@
             if cygwin or pgi or gnu_darwin:
                 mem_val = 'cstdlib'
             else:
-                mem_val = 'jemalloc'
+                mem_val = 'cstdlib'
     else:
         raise ValueError("Invalid flag: '{0}'".format(flag))
     return mem_val
--- a/util/chplenv/chpl_tasks.py
+++ b/util/chplenv/chpl_tasks.py
@@ -29,7 +29,7 @@
                 using_qthreads_incompatible_cce):
             tasks_val = 'fifo'
         else:
-            tasks_val = 'qthreads'
+            tasks_val = 'fifo'
     return tasks_val
 
 
